v2g_liberty.py

```diff
@@ -335,6 +326,17 @@ def set_next_action(self):
 
         return
 
+    def set_chargemode_to_automatic(self):
+        # This is a somewhat clumsy way to set the mode to automatic but HA does not support radio buttons,
```

Comment:
Now these comments can become a docstring, using the `"""` syntax.

Reply:
Will do.